.gitattributes

```diff
@@ -14,7 +14,7 @@ statements_gettextgen.po linguist-generated=true
 
 # The following are marked as binary and generated since they can be
 # easily overwritten and don't need deep review.
-cln-grpc/proto/node.proto -text -diff linguist-generated=true
-cln-grpc/src/convert.rs -text -diff linguist-generated=true
-cln-rpc/src/model.rs -text -diff linguist-generated=true
-contrib/pyln-testing/pyln/testing/node_pb2.py linguist-generated=true
\ No newline at end of file
+#cln-grpc/proto/node.proto -text -diff linguist-generated=true
+#cln-grpc/src/convert.rs -text -diff linguist-generated=true
+#cln-rpc/src/model.rs -text -diff linguist-generated=true
```

Comment:
Please leave the `linguist-generated` annotation in there, this still allows us to expand, but the diff won't load by default, reducing the clutter for reviewers.

Reply:
Sorry about this, it was a mistake